Makefile

```diff
@@ -25,21 +25,36 @@ test: ## Run unit tests. Example: make test
 	go test -race -count 1 ./cmd/... ./pkg/...
 .PHONY: test
 
+# allows overwrite of node registrar image
+NODE_REGISTRAR_IMAGE ?=
+# allows overwite of CSI driver image
+DRIVER_IMAGE ?=
+# non-empty deploy mode will enable read-only mode deployment
+DEPLOY_MODE ?=
+
```

Comment:
1.) Should these have a default specified?
2.) These should be at the top of the file so that anyone using it will know right away what options they have

Reply:
Those are variables previously used on `deploy.sh`, now also exposed by `Makefile` as well.

Reply:
The `DEPLOY_MODE` is new, and I'm updating the comment to make clear what's its objective. Let me know if it's clear.

Reply:
the comment update looks good @otaviof 

but I agree with @coreydaley that it should be moved toward the top, above `all: ....`

same with the TEST_SUITE and TEST_TIMEOUT that I think @adambkaplan recently added

thanks